.github/workflows/checks.yml

```diff
@@ -49,12 +49,26 @@ jobs:
           php-version: 7.4
           coverage: none
 
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache Composer
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: composer-${{ hashFiles('composer.json') }}
```

Comment:
why do you alternate between
`composer-${{ hashFiles('composer.json') }}`
and
`composer-${{ hashFiles('**/composer.json') }}`
?
Anyway, I think you should use this to make the cache additive, since you not commit a `composer.lock`
`composer-${{ github.sha }}`

Reply:
I used `**/composer.json` when running `composer update` against each component individualy
And `composer.json` when running `composer update` on the root directory only

> Anyway, I think you should use this to make the cache additive, since you not commit a composer.lock
composer-${{ github.sha }}
:facepalm: of course... the composerjson does not vary, but the reste of the world move and new version could be available..

thanks!